.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on: [push]
+jobs:
+  run-unit-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install sphinx==3.2.1 pyspark==3.0.1 tensorflow-cpu==1.15.0 pytest
```

Comment:
Update pyspark to 3.2.1? Shell we update other packages too?

Reply:
Sounds good, upgraded all packages to the latest.